schema.ts

```diff
@@ -152,3 +161,17 @@ export const comments = pgTable(
     }),
   ],
 );
+
+export const myChatParticipantsView = pgView("my_chats_participants").as(
```

Comment:
A view filtering the information was needed, because it doesn't seem to be possible to create a `rls` rule that filters it's own table, thus, making rules like "I want to see the chat participants of the chats where I'm also a chat participant" impossible to represent using RLS - it leads to a recursion error.